.circleci/config.yml

```diff
@@ -1490,7 +1495,12 @@ jobs:
                 name: Build release package as a job artifact
                 command: |
                   mkdir -p build
-                  FILENAME=$(npm pack)
+
+                  # Run prepack script manually, instead of providing --unsafe-perm option to npm
+                  yarn prepack
+
+                  FILENAME=$(npm pack | tail -1)
```

Comment:
this is technically unrelated to these changes right? is it a different minor improvement that you want to bring in? or there's some direct reason?